Ricky bug fixes

rossird opened the pull request:
Added bug fixes and error displaying. Also removed duplicate resource folder. Indices and use case views now auto-wrap text.

Known issue: When clicking the manage button and re-updating how to index, the indices and use case view don't get cleared, even though combo box is changed.

rossird committed:
Added error checking on resource directory, acronym file, stop word
file. RequirementsView and IndicesView now auto-wrap text.

rossird committed:
Fixed the error checking for stop words. It was looking at acronym the
be checked.

rossird committed:
Changed Indexer logic to always tokenize, even when not asked so that
the Index can be created correctly and displayed.

rossird committed:
Changed the error checking logic for the acronym and stop word files.
Now displays more detailed errors.

rossird committed:
Added more use case files. Modified GUI to display Resource Directory.
Acronym Converter displays a text box on error. Choosing use case 0 also
clears the requirementIndicesView text.

drakeae commented:
It looks fine to me. The only issue is that I left my quick fix for grabbing whatever address was typed in as the resource directory, so we'll have to handle that when that and Feat 2's merge pull request go through.

Either way, I'll go ahead and update the RTM with these fixes.

rossird committed:
Merge branch 'master' into Ricky-BugFixes

Conflicts:
	src/dialogs/GreetingMsg.java
	src/tracing/views/RequirementsView.java

rossird committed:
Added a check for indicesText in requirementIndicesView.setIndicesText()
(for when it first starts). Seems to have integrated together with
Feature 2 quite well.

rossird commented:
Merged with feature 2. Will merge in about an hour if there are no objections.
